controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java

```diff
@@ -265,10 +268,16 @@ final void startAsync() {
         }
     }
 
-    final void stopAsync() {
+    final void stopAsync(Boolean interruptDelegate) {
```

Comment:
Can we use primitive boolean here as null is not being passed anywhere. That will also reduce additional check Boolean.TRUE.equals(interruptDelegate)

Reply:
done